bb_server/src/command/custom/fly.rs

```diff
@@ -0,0 +1,14 @@
+use crate::command::{Command, CommandTree};
+
+pub struct FlyCommand {}
```

Comment:
why add these empty structs? they don't do anything

Reply:
Because the name of the init function is the same and i then know what Command to init